dapp/src/hooks/useSwapEstimator.js

```diff
@@ -507,6 +520,18 @@ const useSwapEstimator = ({
         isRedeem ? coinToReceiveDecimals : 18
       )
 
+      const decimals = swapMode === 'redeem' || selectedCoin === 'dai' ? 18 : 6
```

Comment:
almost same bit of code happens 3 times. Looks like it could be refactored into a function and called 3 times